app/src/main/res/values-pt/strings.xml

```diff
@@ -162,4 +162,57 @@
     <string name="welcome_message">Instruções:\n1. Insere o nível de treinador\n2. Carrega em START e permite a captura de ecrã\n3. Abre o teu Pokemon na aplicação \'Pokémon Go\'\n4. Carrega no botão IV para iniciar o exame\n5. Ajusta os dados se necessário\n6. Observa o IV!</string>
     <string name="goiv_is_open_source">GoIV é agora em código aberto!\ngithub.com/farkam135/GoIV\nreddit.com/r/GoIV</string>
     <string name="trainer_level">Nível de Treinador</string>
+	
+	<string name="pokemon_name">Nome do Pokémon:</string>
+    <string name="cp">CP</string>
+    <string name="hp">HP</string>
+    <string name="use_the_slider_below_to_align_the_arc">Deslize para alinhar o arco</string>
+    <string name="cancel">Cancelar</string>
+    <string name="check_iv">Verifica o IV</string>
+    <string name="main_not_numeric">%1$s não é um número.</string>
+    <string name="main_invalide_trainerlvl">Nível de treinador inválido!</string>
+    <string name="notification_title">GoIV funcionando - Nível %1$s</string>
+    <string name="notification_text">Toque para abrir</string>
+    <string name="ivtext_title">O teu nível %1$s, CP %2$s, HP %3$s %4$s pode ser:</string>
+    <string name="ivtext_possibilities">%1$s mais possibilidades...</string>
+    <string name="ivtext_no_possibilities">Não há possibilides, por favor verifique os dados!</string>
+    <string name="ivtext_iv">Min: %1$s%% Média: %2$s%% Max: %3$s%%\n</string>
+    <string name="ivtext_evolve">Pode evoluir para %1$s:\n</string>
+    <string name="ivtext_evolve_further">Pode ser ainda mais evoluído para %1$s:</string>
+    <string name="ivtext_many_possibilities">Há muitas possibilidades para este Pokémon. Tente usar o POWER UP!</string>
+    <string name="ivtext_max_lvl_cost">Custo para chegar ao nível %1$s:</string>
+    <string name="ivtext_max_lvl_cost2">Candy: %1$s Stardust: %2$s</string>
+    <string name="main_battery_saver">Poupança de bateria (Pesquiza por captura de ecrã)</string>
+    <string name="ivtext_cp_lvl">CP ao nivél %1$s: %2$s - %3$s</string>
+    <string name="ivtext_stats">Atk: %1$02d Def: %2$02d Sta: %3$02d — %4$3d%%</string>
```

Comment:
I meant—can you drop these strings, like `ivtext_stats`, that are unchanged compared to the original translation?